style/README.md

```diff
@@ -39,193 +40,3 @@ Naming
-  instead to emphasize code branches.
-* Avoid explicit return statements.
-* Avoid using semicolons.
-* Avoid assignments on conditionals.
-* Prefer `detect` over `find`.
-* Prefer `inject` over `reduce`.
-* Prefer `map` over `collect`.
-* Prefer `select` over `find_all`.
-* Prefer single quotes for strings.
-* Use `do...end` for multi line blocks and `{...}` for one line blocks.
-* Use `_` for unused block parameters.
-* Use `%{}` for single-line strings needing interpolation and double-quotes.
-* Use `&&` and `||` for Boolean expressions.
-* Use `?` suffix for predicate methods.
-* Use `CamelCase` for classes and modules, `snake_case` for variables and
-  methods, `SCREAMING_SNAKE_CASE` for constants.
-* Use `def self.method`, not `def Class.method` or `class << self`.
-* Use `def` with parentheses when there are arguments.
-* Use `each`, not `for`, for iteration.
-* Use heredocs for multi-line strings.
-* Use `%w()` and `%i()` for hard-coded lists of strings and symbols.
-
-ERb
----
-
-* Use Slim
-
-Slim
-----
-[Sample](/groupbuddies/guides/blob/master/style/samples/slim.rb)
-
-* When wrapping long lines, keep the method name on the same line as the Slim
-  interpolation operator and keep each method argument on its own line.
-* Prefer double quotes for attributes.
-
-HTML
-----
-
-* Prefer double quotes for attributes.
-
-Rails
------
-
-* Avoid `member` and `collection` routes.
-* Use private instead of protected when defining controller methods.
-* Name date columns with `_on` suffixes.
-* Name datetime columns with `_at` suffixes.
-* Name initializers for their gem name.
-* Order ActiveRecord associations alphabetically by attribute name.
-* Order ActiveRecord validations alphabetically by attribute name.
-* Order controller contents: filters, public methods, private methods.
-* Order model contents: constants, macros, public methods, private methods.
-* Put application-wide partials in the [`app/views/application`] directory.
-* Use `def self.method`, not the `scope :method` DSL.
-* Use the default `render 'partial'` syntax over `render partial: 'partial'`.
-
-[`app/views/application`]: http://asciicasts.com/episodes/269-template-inheritance
-
-Rails Routes
-------------
-
-* Avoid the `:except` option in routes.
-* Order resourceful routes alphabetically by name.
-* Use the `:only` option to explicitly state exposed routes.
-
-Background Jobs
----------------
-
-* Define a `PRIORITY` constant at the top of delayed job classes.
-* Define two public methods: `self.enqueue` and `perform`.
-* Put delayed job classes in `app/jobs`.
-
-Email
------
-
-* Use one `ActionMailer` for the app. Name it `Mailer`.
-* Use the user's name in the `From` header and email in the `Reply-To` when
-  [delivering email on behalf of the app's users].
-
-[delivering email on behalf of the app's users]: http://robots.thoughtbot.com/post/3215611590/recipe-delivering-email-on-behalf-of-users
-
-Testing
```

Comment:
@zamith @naps62 This section is mostly ruby/rspec specific. Should we move this under ruby or something like that?